prov/gni/src/gnix_hashtable.c

```diff
@@ -715,9 +724,6 @@ int _gnix_ht_remove(gnix_hashtable_t *ht, gnix_ht_key_t key)
 
 void *_gnix_ht_lookup(gnix_hashtable_t *ht, gnix_ht_key_t key)
 {
-	if (ht->ht_state != GNIX_HT_STATE_READY)
-		return NULL;
-
```

Comment:
Maybe what I'm wondering is if anyone expects this behavior.  Even if it's stupid, we could replace this with a raw assert to help with our own debugging.